packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -1103,23 +1103,23 @@ describe('RPC Server Acceptance Tests', function () {
                     Assertions.feeHistory(res, {
                         resultCount: blockCountNumber,
                         oldestBlock: oldestBlockNumberHex,
-                        chechReward: true
+                        checkReward: true
                     });
 
-                    expect(res.baseFeePerGas[0]).to.equal(datedGasPriceHex);
+                    expect(res.baseFeePerGas[1]).to.equal(datedGasPriceHex);
```

Comment:
Q: what's the logic here? Why does it need to be the 2nd index?
Please add a comment so it's clear

Reply:
In the before hook, we are feeding default fees to the mirror node. So that we know how many blocks we should take and where to expect those fees. But if there is discrepancy in the blocks, the first element in this array is taken from the consensus node, not from the mirror node. This is the reason sometimes this value in the first index is wrong.

The other test was similar. By the time we're requesting the last block, it's already ahead by one. So we were excepting it to be 10 ahead, but it's actually 11 ahead on rare occasions.

Reply:
Okay. Yeah just add a summarized comment above the code line so there's context.
Also nice catch :)